CHANGES.rst

```diff
@@ -6,6 +6,13 @@ assign_wcs
 
 - Use isinstance instead of comparison with a type for lamp_mode inspection [#7801]
 
+associations
+------------
+
+- Changes the way associations are create for field  ``FXD_SLIT``.  Assocations
+  for this field will now include values for both ``S200A1`` and ``S200A2``,
+  instead of separating them. [#7795]
```

Comment:
```suggestion
- Update the Level 3 rules for NIRSpec fixed-slit observations so that sets of
  exposures using ``FXD_SLIT=S200A1`` and ``S200A2`` will be placed into
  the same ASN.  [#7795]
```